src/databricks/labs/remorph/reconcile/connectors/oracle.py

```diff
@@ -1,56 +1,61 @@
-from pyspark.errors import PySparkException
-from pyspark.sql import DataFrame, DataFrameReader
+from pyspark.sql import DataFrame, DataFrameReader, SparkSession
+from sqlglot import Dialects
 
 from databricks.labs.remorph.reconcile.connectors.data_source import DataSource
+from databricks.labs.remorph.reconcile.connectors.jdbc_reader import JDBCReaderMixin
+from databricks.labs.remorph.reconcile.connectors.secrets import SecretsMixin
 from databricks.labs.remorph.reconcile.constants import SourceDriver, SourceType
 from databricks.labs.remorph.reconcile.recon_config import JdbcReaderOptions, Schema
+from databricks.sdk import WorkspaceClient
 
 
-class OracleDataSource(DataSource):
+class OracleDataSource(DataSource, SecretsMixin, JDBCReaderMixin):
+
+    def __init__(
+        self,
+        engine: Dialects,
```

Comment:
It should be `DialectType` instead of `Dialects`.

Reply:
we are using SQLGLOT_DIALECTS which is of type Dialects.

Reply:
good eagle eye catch by @vijaypavann-db  but it should be `Dialect` not `DialectType`

Refer `_get_dialect` function under `config.py`

Reply:
It may cause issues when we add `mypy` checks